toqito/helper/npa_hierarchy.py

```diff
@@ -128,7 +128,7 @@ def npa_constraints(  # pylint: disable=too-many-locals
     assemblage: dict[tuple[int, int], cvxpy.Variable], k: int | str = 1, referee_dim: int = 1
 ) -> list[cvxpy.constraints.constraint.Constraint]:
     r"""
-    Generate the constraints specified by the NPA hierarchy up to a finite level.
+    Generate the constraints specified by the NPA hierarchy up to a finite level :cite:`Navascu_s_2008`.
```

Comment:
Hmm, that's a weird bibtex key. Maybe if we follow the prior comment and uniformize these things, the weirdness here will go away though, so this comment might just be moot.

Reply:
Yep, this was the key from arxiv's bib info. I was lazy here as well.